source/extensions/clusters/dynamic_forward_proxy/BUILD

```diff
@@ -2,16 +2,17 @@ licenses(["notice"])  # Apache 2
 
 load(
     "//bazel:envoy_build_system.bzl",
-    "envoy_cc_library",
+    "envoy_cc_extension",
     "envoy_package",
 )
 
 envoy_package()
 
-envoy_cc_library(
+envoy_cc_extension(
     name = "cluster",
     srcs = ["cluster.cc"],
     hdrs = ["cluster.h"],
+    security_posture = "robust_to_untrusted_downstream",
```

Comment:
How do you want to handle something where this is what I would consider an alpha feature? I assume we will do a follow up here about feature status also potentially?

Reply:
Yeah, we should have an independent maturity field. This should be combined with security posture to give an idea of how trusted the extension really is. I'm happy to do it in this PR if I don't have to exercise too much value judgement; what about picking all extensions that only have alpha APIs and maybe you can give me any extras?

What should the maturity levels be? [alpha, stable]?

Reply:
I have now added a WiP and alpha status tag, PTAL.